protogen/api/auditlog/v1/message.pb.go

```diff
@@ -0,0 +1,1736 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: api/auditlog/v1/message.proto
+
+package auditlog
```

Comment:
why is the auditlog proto go file added as part of this?

Reply:
because it came with syncing to the latest protos